ui/app/clinical/consultation/views/treatmentSections/drugOrderHistory.html

```diff
@@ -19,10 +19,36 @@
 					<div class="inpage-tab-content">
                         <p ng-if="!drugOrderGroup.drugOrders.length" class="placeholder-text">{{ ::'MEDICATION_NO_RECENT_TREATMENT' | translate }}</p>
                         <ul class="table-mimic">
-							<li id="ordered-drug-orders" ng-repeat="drugOrder in drugOrderGroup.drugOrders" ng-class="{'active-drug' : drugOrder.isActive(), 'highlight' : drugOrder.isBeingEdited, 'scheduled-drug': drugOrder.isScheduled()}">
+							<li id="ordered-drug-orders" bm-pop-over autoclose="true" ng-repeat="drugOrder in getPreviousDrugAlerts(drugOrderGroup.drugOrders)" ng-class="{'active-drug' : drugOrder.isActive(), 'highlight' : drugOrder.isBeingEdited, 'scheduled-drug': drugOrder.isScheduled()}">
                                 <div class="drug-details" ng-class="{'strike-through': drugOrder.isDiscontinuedOrStopped()}">
-                                    <div class="drug-dosage">
-                                        <strong class="drug-name">{{drugOrder.getDisplayName()}}</strong>
+                                    <div class="drug-dosage relative">
+                                        <strong class="drug-name popover-trigger">
```

Comment:
Consider adding in separate angular directive, the modal. And ensure the string `dismissed` comes from translation file.